shm/Map.cpp

```diff
@@ -1,6 +1,31 @@
 #include "Map.hpp"
-
 #include <algorithm>
+#include <random>
+#include <cstddef>
+
+Map::Map(){
+    islandsLocations_.reserve(amountOfIslands);
+    generateIslands(amountOfIslands);
+}
```

Comment:
@PierwszyKrytykRP  I can see it's fine in the newest version.

Reply:
Yea i see it now too. Maybe something with cache